tests/views/tp.py

```diff
@@ -144,13 +144,16 @@ def _test_browse_view(tp, request, response, kwargs):
     view_context_test(ctx, **assertions)
     if vfolders:
         for vfolder in ctx["vfolders"]["items"]:
-            assert vfolder["is_grayed"] is False
+            assert (vfolder["is_grayed"] and not ctx["is_admin"]) is False
```

Comment:
@unho, I couldn't understand why we have `False` always here. It worked for Anonymous users but not for admins.
This change takes into account that we show non-public vftis to admins.